lang/post-worktime-logger-it_IT.po

```diff
@@ -0,0 +1,183 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Post Worktime Logger v1.2.3\n"
```

Comment:
This seems to be the wrong version number? Did you checked out swimlane/1.4.0 or master?